docs/GettingStartedDocs/GettingStarted.Windows.md

```diff
@@ -79,7 +78,7 @@ C:\Program Files\LLVM\bin\ld.lld.exe
 OCaml
 ---------------------------------
 Install [OCaml for Windows (64-bit)](https://www.ocamlpro.com/pub/ocpwin/ocpwin-builds/ocpwin64/20160113/).
-Please download and install the mingw64 exe for OCaml, for example, ocpwin64-20160113-4.02.1+ocp1-mingw64.exe.
+Please download and install the mingw64 exe for OCaml, for example, https://www.ocamlpro.com/pub/ocpwin/ocpwin-builds/ocpwin64/20160113/ocpwin64-20160113-4.01.0+ocp1-mingw64.exe.
```

Comment:
Is there a reason the version was dropped from:
> ocpwin64-20160113-4.**02.1**+ocp1-mingw64.exe 

To:

> ocpwin64-20160113-4.**01.0**+ocp1-mingw64.exe

I don't believe it matters functionally, but we should stick with what was previously tried by @anakrish. IIRC, the primary emphasis in this doc was on taking a mingw64 version of the install, not the msvc64 versions.

Reply:
Good catch @CodeMonkeyLeet . I was the last one to update the Windows Starting guide with approval from @anakrish .
@jxyang - Please do not bump down the revision number pointed to. 4.02.1 is the one that we are using. Did you try this version? If not, please use this.